library-config.yaml

```diff
@@ -1,4 +1,4 @@
-artifactVersion: 0.5.1
+artifactVersion: 0.5.2
```

Comment:
This doesn't need to be updated. This number is updated for Feature additions without breaking changes. We are not adding any feature additions here.... Just a utility function.

Reply:
ah i did it because i saw it in the readme [here](https://github.com/simplanio/simplan-framework?tab=readme-ov-file#build-and-publish-to-artifactory). For my understanding, what's the difference between a feature and a utility function?